src/SearchResults.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import moment from "moment";
+import CustomerProfile from "./CustomerProfile";
+import NewBookings from "./NewBookings";
+
+
+const getDaysDifference = (date1, date2) => {
+  let a = moment(date1);
+  let b = moment(date2);
```

Comment:
Think we can use `const` here? Always better to use const when we are not going to be reassigning a variable 😄